src/Sulu/Bundle/AdminBundle/Resources/js/components/Popover/Popover.js

```diff
@@ -38,19 +38,26 @@ class Popover extends React.Component<Props> {
     };
 
     @observable popoverChildRef: ElementRef<*>;
-
     @observable popoverWidth: number;
-
     @observable popoverHeight: number;
 
-    componentDidMount() {
+    mutationObserver: MutationObserver;
+
+    constructor(props: Props) {
+        super(props);
+
         window.addEventListener('blur', this.close);
         window.addEventListener('resize', this.close);
+        this.mutationObserver = new MutationObserver(() => {
+            this.setPopoverSize(0, 0);
```

Comment:
i guess this is necessary somehow? asking because the `updateDimensions()` method will call `setPopoverSize()` with some calculated parameters anyway

Reply:
Yeah, the problem is that if we don't reset the popover size first, there will already be some width and height defined on that element, and so it will still not grow. Can also add a comment explaining that.

Reply:
okay, that would be nice! 🙂